### DISCUSSION TITLE
Full node 1.14.4 confirmation

### DISCUSSION THREAD

**Discussion by victorsk2019** [dogecoin/dogecoin#2638] [General]
Hi everyone,

I've set up a full node 1.14.4 on a dedicated server I'm running (https://stage4linux.com/) and used RPC calls to retrieve some information about its status:

![image](https://user-images.githubusercontent.com/47955466/138195205-e15fbe56-b477-4de7-8181-a5074eb48946.png)

Please let me know if the values showing on the above snapshot are correct for this full node version.  The last thing I want is to have a misconfigured full node service running 😉 so please let me know if, for example, "paytxfee" value should be something else?

Thanks,
Victor.

**Comment by patricklodder** [comment]
`paytxfee` is a wallet configuration, so that doesn't matter unless you have a wallet you're using on there (I personally disable the wallet on nodes that do not need it with `disablewallet=1`)

Keep an eye on `score`. It should grow over time. If it stalls for a longer time, you may have connectivity issues

**Reply by victorsk2019** [reply]
Thank you 👍

**Comment by victorsk2019** [comment]
>    I personally disable the wallet on nodes that do not need it

I've modified ebuilds in my Gentoo PR gentoo/gentoo#22229 for building with wallet disabled (before wallet was required part of build).
